packages/transformers/kotlin/src/KotlinTransformer.js

```diff
@@ -1,11 +1,57 @@
 // @flow
 
 import {Transformer} from '@parcel/plugin';
+import fs from '@parcel/fs';
```

Comment:
I think you don't need `fs` from `@parcel/fs` you can use `asset.fs`.

Reply:
(Correct, you *have* to use `asset.fs`)